pages/common/git-repl.md

```diff
@@ -0,0 +1,21 @@
+# git repl
+
+> Open a Git shell-like command line.
```

Comment:
How about that:
```suggestion
> Git REPL (read-print-evaluate-loop) - an interactive Git shell. 
```
?

Reply:
`evaluate` should come before `print`, and I've mostly seen `evaluate` shortened to `eval` 🤔

Reply:
Not thrilling, but seems good to me.

Reply:
Im going to edit the suggestion